features/createroom/impl/build.gradle.kts

```diff
@@ -43,7 +43,7 @@ dependencies {
 
     implementation(projects.libraries.core)
     implementation(projects.libraries.architecture)
-    implementation(projects.libraries.matrix.api)
+    api(projects.libraries.matrix.api)
```

Comment:
Why are you changing to api there? same for all other implementation->api changes.